### PR TITLE
clean up kv metric names

### PR DESCRIPTION
**Pull request by mziolekda**
Make all the kvutils committer metrics to follow the pattern of:
kvutils.committing.\<committerName\>.*

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags, if appropriate
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/PackageCommitter.scala [hunk 1]
```diff
@@ -162,6 +162,8 @@ private[kvutils] case class PackageCommitter(engine: Engine)
     "buildLogEntry" -> buildLogEntry
   )
 
+  override lazy val committerName = "packageUpload"
```
Comment (dajmaki) [inline ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/PackageCommitter.scala]: make it "package_upload" to follow normal conventions.

File: ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/PartyAllocationCommitter.scala [hunk 2]
```diff
@@ -161,4 +161,6 @@ private[kvutils] case object PartyAllocationCommitter
     "buildLogEntry" -> buildLogEntry
   )
 
+  override lazy val committerName = "partyAllocation"
```
Comment (dajmaki) [inline ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/PartyAllocationCommitter.scala]: `party_allocation`

File: ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/PackageCommitter.scala (hunk 1 above)
Comment (mziolekda) [reply ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/PackageCommitter.scala]: Done

File: ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/PartyAllocationCommitter.scala (hunk 2 above)
Comment (mziolekda) [reply ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/PartyAllocationCommitter.scala]: Done

**Pull request merged**